src/main/java/com/fcgl/madrid/forum/dataModel/Comment.java

```diff
@@ -17,13 +17,13 @@
     private long updatedDate;
     private int likes;
     private int dislikes;
-    private int userId;
+    private Integer userId;
 
     @ManyToOne()
     @JsonBackReference
     private Post post;
 
-    public Comment(String message, Post post, int userId) {
+    public Comment(String message, Post post, Integer userId) {
```

Comment:
We should make it so this class throws an Exception is any of these are null.

This request should throw an Exception because we don't have a message or a post in the request
`curl -X POST -d userId=1 http://localhost:8082/forum/comment/v1/postComment`

The expected response being `{"code":2,"httpCode":400,"messages":["message must not be null","post must not be null"]}`

You can do this by adding `@NotNull` param, this PR has an example
https://github.com/fcgl/forum/pull/7/files#diff-4d63ecd7238cb1a1e6ceec184f8e1af3R21

Reply:
Done